components/ActionSheet/tests/unit/MDCActionSheetThemeTest.m

```diff
@@ -105,6 +106,31 @@ - (void)testApplyThemerWithOnlyMessage {
 
 #pragma mark - default test
 
+- (void)testApplyThemer {
+  // Given
+  MDCActionSheetController *tempActionSheet = [[MDCActionSheetController alloc] init];
+  MDCActionSheetScheme *scheme = [[MDCActionSheetScheme alloc] init];
+  scheme.colorScheme = self.colorScheme;
+  scheme.typographyScheme = self.typographyScheme;
+
+  // When
+  [MDCActionSheetThemer applyScheme:scheme toActionSheetController:tempActionSheet];
+  [MDCActionSheetColorThemer applySemanticColorScheme:self.colorScheme
+                              toActionSheetController:self.actionSheet];
+  [MDCActionSheetTypographyThemer applyTypographyScheme:self.typographyScheme
+                                toActionSheetController:self.actionSheet];
```

Comment:
"applySemanticColorScheme" and "applyTypographyScheme" could potentially hide issues in "applyScheme" from being fully tested.
These could be tested in a separate test, or altogether skipped, since their logic is being tested when called from applyScheme.

Reply:
Are you saying there could be an issue in applySemanticColorScheme/applyTypographyScheme that isn't caught?